chunky/src/java/se/llbit/chunky/ui/render/SkyTab.java

```diff
@@ -102,9 +102,9 @@ public SkyTab() throws IOException {
     cloudZ.onValueChange(value -> scene.sky().setCloudZOffset(value));
 
     fogDensity.setTooltip("Fog thickness. Set to 0 to disable volumetric fog effect.");
-    fogDensity.setRange(0, 2);
-    fogDensity.clampMin();
+    fogDensity.setRange2(0, 1);
```

Comment:
Why not add `sliderMin` as third argument to `setRange`?

Reply:
From my limited research optional args aren't really supported by Java which would have been my choice. There is the option of using an overload but I wasn't sure which one would be preferred.

Reply:
Either add another method with a third argument all call that one from the method with two arguments. Or just remove the old one and change all places where it is used (probably not that many in Chunky).

Reply:
While it is only 33 occurrences, given we only need the additional parameter for this one case I do believe using the overload is the best choice. I'll make the necessary changes such that we have a method setRange(min, max, SliderMin) and I'll remove the setRange2().